app/coffeescript/components/mixins/stored_markers.coffee

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+define [
+  'jquery'
+  'flight/lib/component'
+], (
+  $
+  defineComponent
+) ->
+
+  storedMarkers = ->
+
+    @defaultAttrs
+      storageKey: 'viewedMapMarkers'
+
+    @load = ->
+      return {} unless Storage?
```

Comment:
The browser base on this http://stackoverflow.com/questions/11214404/how-to-detect-if-browser-supports-html5-local-storage